service/publishing.gradle

```diff
@@ -1,5 +1,8 @@
 jib {
 	container {
+		to {
```

Comment:
Somehow there's a magic `extractModuleInfo` task that artifactory called that set the proper image name.  This appears to accomplish the same thing.